CHANGES.md

```diff
@@ -28,6 +28,12 @@ breaking changes, and mappings for the large list of deprecated functions.
 
 [Migration guide]: https://github.com/openssl/openssl/tree/master/doc/man7/migration_guide.pod
 
+### Changes between 3.0.10 and 3.0.10+quic [1 Aug 2023]
+
+* Add QUIC API support from BoringSSL
+
+   *Todd Short*
+
```

Comment:
This seems to be missing the latest version numbers here...

Reply:
Let me close in favor of yours: I think I bobbled this by starting from 3.0.10.